src/lit-element.ts

```diff
@@ -68,10 +67,10 @@ export class LitElement extends UpdatingElement {
    * optimizations. See updating-element.ts for more information.
    */
   protected static['finalized'] = true;
+
   /**
-   * Render method used to render the lit-html TemplateResult to the element's
-   * DOM.
-   * @param {TemplateResult} Template to render.
+   * Render method used to render the value to the element's DOM.
+   * @param {unknown} The value to render.
```

Comment:
These need the property names as the first word following the type.

Reply:
Fixed.